# Slides - SW03

In [1]:
import pandas as pd
import numpy as np
from pandas import Series, DataFrame
import matplotlib.pyplot as plt
import seaborn as sb
import scipy.integrate as integrate
import scipy.special as special
from scipy.stats import uniform, expon, norm

## Uniforme Verteilungen

$ P(X ≤ 1) $

In [2]:
uniform.cdf(x=1, loc=0, scale=7)

0.14285714285714285

$ P(0.5 ≤ X ≤ 2.2) $

In [3]:
uniform.cdf(x=2.2, loc=0, scale=7) - uniform.cdf(x=0.5, loc=0, scale=7)

0.2428571428571429

In [4]:
uniform.pdf(x=1, loc=0, scale=7)

0.14285714285714285

In [5]:
uniform.rvs(size=3, loc=0, scale=7)

array([6.86120332, 6.1504072 , 0.63329744])

## Exponential-Verteilungen

In [6]:
expon.cdf(x=4, loc=0, scale=1/3)

0.9999938557876467

In [7]:
expon.pdf(x=1, loc=0, scale=1/3)

0.14936120510359185

## Normalverteilung

Wie gross ist die W'keit, dass jemand einen IQ von 130 und mehr hat?

Mittelwert = 100

Standardabweichung: 15

In [8]:
1 - norm.cdf(x=130, loc=100, scale=15)

0.02275013194817921

Die W'keit das eine Person einen IQ von 130 und mehr hat, ist 2.2%.

In welchem Intervall liegen 90% der IQ Ergebnisse?

In [9]:
norm.ppf(q=0.05, loc=100, scale=15)

75.32719559572791

In [10]:
norm.ppf(q=0.95, loc=100, scale=15)

124.67280440427209

Das Intervall der Personen liegt zwischen 75.33 <> 124.67.

Sei X ∼ N(µ, σ 2 ) mit µ = 2 und σ 2 = 4. Berechnen Sie P(X ≤ 5).

In [11]:
norm.cdf(x=1.5)

0.9331927987311419